buildprocess/configureWebpack.js

```diff
@@ -286,12 +287,21 @@ function configureWebpack(terriaJSBasePath, config, devMode, hot, MiniCssExtract
         })
     );
     config.plugins.push(
-        new ForkTsCheckerNotifierWebpackPlugin({
-            excludeWarnings: true,
-            // probably don't need to know first check worked as well - disable it
-            skipFirstNotification: true
-        })
-    );
+      new ForkTsCheckerNotifierWebpackPlugin({
+          excludeWarnings: true,
+          // probably don't need to know first check worked as well - disable it
+          skipFirstNotification: true
+      })
+  );
+
+  // Copy assimpjs.wasm file - this is used by AssImpCatalogItem (see AssImpCatalogItem.forceLoadMapItems())
+  config.plugins.push(
+    new CopyPlugin({
```

Comment:
We are currently using Webpack v4. I had a little go but couldn't get it working. I'm not that familiar with webpack though. It would be good to get @na9da or @steve9164 opinion on this though

Reply:
I'm going to leave this as is - we can tackle this when we upgrade webpack - or move to something else